Makefile

```diff
@@ -104,4 +106,50 @@ compare_trace: $(CAIRO_RS_TRACE) $(CAIRO_TRACE)
 compare_memory: $(CAIRO_RS_MEM) $(CAIRO_MEM)
 	cd tests; ./compare_vm_state.sh memory
 	
-	
+compare_memory_devnet:
+# Set up the virtual envs
+	scripts/memory_comparator/build_envs.sh
+# Clone the starknet-devnet from github
+	git clone git@github.com:Shard-Labs/starknet-devnet.git
```

Comment:
Let's checkout a fixed tag instead of `main`. We need stability for comparisons.